### DISCUSSION TITLE
Database upgrade seems to have failed

### DISCUSSION THREAD

**Discussion by DeveTho** [supabase/supabase#19081] [Questions]
Last week, I upgraded the Postgres version through "Project Settings > Infrastructure" from 15.1.0.131 to 15.1.0.133. I got a notification that something went wrong, though when I looked further, everything seemed to work and the Infrastructure page is saying I'm on 15.1.0.133.

However, on the dashboard, I keep getting the notification that the upgrade was not successful and that I should contact support.
![image](https://github.com/supabase/supabase/assets/7809659/e26f55ac-10ba-4899-8993-a9806c7c0e44)

And I'm noticing that after the upgrade, the size of my database has dropped by 50 MB (which is more than half of its size), so I guess something didn't go completely right?

Anyone knows what could've gone wrong and what I could do to fix it? Or can this only be looked into by Supabase support (I did send a support request, but I'm on the free tier)?  
Thanks a lot for any replies! :)

**Comment by GaryAustin1** [comment]
If you have tried restarting the database then you'll have to wait on support.

**Reply by DeveTho** [reply]
Thanks for the tip! Not sure if I did try a restart earlier.

I just did one, though everything seems the same still (notification is still there, database has the same smaller size).

**Comment by DeveTho** [comment]
Supabase support seems to be busy with it, so I guess I'll close this here. :)
